projects/plugins/jetpack/modules/firewall.php

```diff
@@ -0,0 +1,90 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Module Name: Firewall
+ * Module Description: Protect your site with Jetpack's Web Application Firewall
+ * Sort Order: 5
```

Comment:
Any framework of rhyme or reason I should be considering when picking this number?